inlang/source-code/paraglide/paraglide-js/src/cli/commands/compile.ts

```diff
@@ -13,16 +14,25 @@ export const compileCommand = new Command()
 	.action(async (options: { project: string; outdir: string }) => {
 		consola.info(`Compiling inlang project at "${options.project}".`)
 
-		const path = resolve(process.cwd(), options.project)
-
+		const settingsFilePath = resolve(process.cwd(), options.project)
 		const project = exitIfErrors(
 			await loadProject({
-				settingsFilePath: path,
-				//@ts-ignore
+				settingsFilePath,
 				nodeishFs: fs,
+				_capture(id, props) {
+					telemetry.capture({
+						// @ts-ignore the event types
+						event: id,
+						properties: props,
+					})
+				},
 			})
 		)
 
+		telemetry.capture({
+			event: "Paraglide-js compile executed",
```

Comment:
We use all caps for the app. 

```ts
"PARAGLIDE-JS compile executed"
```

Reply:
If you add the capture in the CLI command, you will have to re-implement the the capture in every code that calls the `compile` function (vite plugin, etc.)

A better solution might be to call `PARAGLIDE-JS compile executed` in the compile function itself.